submissions/Oleksii_Drohachov/OOP-exercise/index.js

```diff
@@ -0,0 +1,99 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  friendsToString() {
+    return this.friends.join(", ");
+  }
+
+  output() {
+    const { name, species, gender, saying } = this;
+
+    return `${name}; ${species}; ${gender}; ${saying}; ${this.friendsToString()}`;
+  }
+}
+
+class Mammal extends Creature {
+  constructor(species, name, gender, saying, friends, legs) {
+    super(species, name, gender, saying, friends);
+    this.legs = legs;
+  }
+  output() {
+    return `${super.output()}; ${this.legs}`;
+  }
+}
```

Comment:
`Mammal` is the only class that extends `Creature`. Why do we need `Mammal`?
KISS